server/src/infra/repositories/person.repository.ts

```diff
@@ -96,14 +96,20 @@ export class PersonRepository implements IPersonRepository {
     return this.personRepository.findOne({ where: { id: personId } });
   }
 
-  getByName(userId: string, personName: string): Promise<PersonEntity[]> {
-    return this.personRepository
+  getByName(userId: string, personName: string, withHidden: boolean): Promise<PersonEntity[]> {
```

Comment:
You can add default value to the function's arguments like so


```suggestion
  getByName(userId: string, personName: string, withHidden: boolean = false): Promise<PersonEntity[]> {
```

Reply:
Indeed we don't need `|| false`, `withHidden` has already false as the default parameter

https://github.com/immich-app/immich/blob/48ad55a6e5c6c1538bbb90142219df729dfadec9/server/src/domain/search/dto/search.dto.ts#L89-L97